linux_os/guide/system/auditing/auditd_configure_rules/audit_execution_selinux_commands/audit_rules_execution_rm/rule.yml

```diff
@@ -0,0 +1,49 @@
+documentation_complete: true
+
+prodtype: sle12
+
+title: 'Record Any Attempts to Run rm'
```

Comment:
This rule shouldn't be in `audit_execution_selinux_commands` group, it should have its own group.

I guess it would be okay to have it directly in `auditd_configure_rules` if there are no other similar rules.

Reply:
Thanks, don't see a conflict; so moving to:
linux_os/guide/system/auditing/auditd_configure_rules/audit_rules_execution_rm

For my own information, do rules need to be globally unique? I thought they, did because the stig.profile only contains the rule name and not a path, but I wanted to make sure my understanding was correct.

Reply:
Yeah, the rules should have a global unique ID.